package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cypress",
   "version": "10.6.0",
-  "description": "Cypress.io end to end testing tool",
+  "description": "Cypress.io End-to-End and Component testing tool",
```

Comment:
I'm not against this, but wouldn't adding a description and keywords to the `cli/package.json` make more sense?

Reply:
pulled from https://docs.cypress.io/guides/overview/why-cypress#In-a-nutshell
> Cypress.io front end testing tool built for the modern web. 

Might reach out to marketing about updating the cypress.io home page which called out
> A complete end-to-end testing experience.